.github/workflows/github-page-publish.yml

```diff
@@ -26,9 +26,12 @@ jobs:
           npm run predoc
           npm run build:docs
 
-      - name: Deploy 🚀
-        uses: peaceiris/actions-gh-pages@v3
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./docs
-          enable_jekyll: true
+      - name: Github REST API Call 🛎️
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PARENT_REPO: yorkie-team/yorkie-team.github.io
+          PARENT_BRANCH: main
+          WORKFLOW_ID: <9999999>
```

Comment:
we should get this workflow_id through the following command
curl -X GET -H "Authorization: token $CI_TOKEN" https://api.github.com/repos/$PARENT_REPO/actions/workflows

Reply:
If we edit this Action, I don't think it will build markdown anymore in the `gh-pages` branch. Where is the markdown created after this PR?

Reply:
I was thinking about pushing `docs` directory which is added in `.gitignore` file. but I changed my mind to using gh-pages branch. So i revert deleted codes regarding of deploying gh-pages.

And could you share with me `workflow_id`? Because I don't have access to see `GITHUB_TOKEN` for the current repository, i couldn't run this command. Thanks!

```
curl -X GET -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/$PARENT_REPO/actions/workflows
```